wagtailio/areweheadlessyet/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class AreweheadlessyetConfig(AppConfig):
```

Comment:
Shouldn’t this be:

```suggestion
class AreWeHeadlessYetConfig(AppConfig):
```

to match what you’ve used in the models / follow Django conventions?